java/code/src/com/suse/manager/webui/controllers/DownloadController.java

```diff
@@ -481,10 +482,10 @@ private static void validateToken(String token, String channel, String filename)
             });
         }
         catch (InvalidJwtException | MalformedClaimException e) {
-            log.info(String.format("Forbidden: Token is not valid to access %s in %s: %s",
-                    filename, channel, e.getMessage()));
-            halt(HttpStatus.SC_FORBIDDEN,
-                 String.format("Token is not valid to access %s in %s: %s", filename, channel, e.getMessage()));
+            log.info(String.format("Forbidden: Token %s is not valid to access %s in %s: %s",
+                    token, filename, channel, e.getMessage()));
+            halt(HttpStatus.SC_FORBIDDEN, String.format("Token %s is not valid to access %s in %s: %s",
+                    token, filename, channel, e.getMessage()));
```

Comment:
printing the token in the logfile is ok. But I am not sure about disclosing it on the web error page.
As the token is anyway "invalid" it might be ok as well.

If you think to write it in the logfile is enough, please change this. Otherwise I am also ok with this.

Reply:
+1. I have removed it from error page.